apps/src/sites/studio/pages/levels/_level_group.js

```diff
@@ -152,9 +152,12 @@ function initLevelGroup(levelCount, currentPage, lastAttempt) {
     const isSurvey =
       appOptions.level.anonymous === true ||
       appOptions.level.anonymous === 'true';
+    const isAssessment =
+      appOptions.level.submittable === true ||
+      appOptions.level.submittable === 'true';
     title = isSurvey ? i18n.submitSurvey() : i18n.submitAssessment();
 
-    if (!isSurvey && validCount !== requiredCount) {
+    if (isAssessment && validCount !== requiredCount) {
```

Comment:
I'm having trouble tracking the logic changes here, because I don't fully understand the relation between level.anonymous and level.submittable. I mostly flag this in case the test breaking is a case of real breakage (not just the test needs updating).

Are level.anonymous=true ALWAYS also level.submittable=false?

Reply:
Thanks for this question! Helped me clarify how I wanted to structure this logic and make it more clear. Adding in the new property allows it to be more clear and stick with the frame work of "if it's not a survey, and now also if it's not an activityguidelevel, then it's an assessment" which handles cases of `anonymous` and `submittable` being undefined